src/main/scala/org/broadinstitute/dsde/firecloud/filematch/strategy/OntSingleReadStrategy.scala

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.dsde.firecloud.filematch.strategy
+
+import org.broadinstitute.dsde.firecloud.filematch.result.{FailedMatchResult, FileMatchResult, SuccessfulMatchResult}
+import OntSingleReadStrategy.FILE_CONTAIN_ENDINGS
+
+import java.nio.file.{Path, Paths}
+import scala.util.matching.Regex
+
+class OntSingleReadStrategy extends FileRecognitionStrategy {
```

Comment:
Could we simplify all of this? The `Map[String, Regex]` and the object all feel a bit complicated considering we're just trying to find out if the regex matches the filename and then capturing part of that match. 

We're always matching on the same regex, can we use that to see if the filename matches and to capture the id?

Reply:
Done: https://github.com/broadinstitute/firecloud-orchestration/pull/1581/commits/e97d1d1ef61c78ebb10ad5b46e46a1c200c31c1a